x-pack/solutions/security/plugins/security_solution/public/entity_analytics/components/privileged_user_monitoring/components/key_insights_panel/active_privileged_users_tile/esql_query.tsx

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getPrivilegedMonitorUsersJoin } from '../../../helpers';
+
+export const getActivePrivilegedUsersEsqlCount = (namespace: string) => {
+  return `FROM logs-* METADATA _id, _index
+      ${getPrivilegedMonitorUsersJoin(namespace)}
+      | STATS COUNT(*)`;
```

Comment:
This query needs to be adjusted. Right now, what it's doing is returning a count of all of the events within the time range. However, what we really need is the count of unique `user.name`s within that range.
Luckily, there's an extremely simple way to do this in ES|QL, suggestion below:

```suggestion
  return `FROM logs-* METADATA _id, _index
      ${getPrivilegedMonitorUsersJoin(namespace)}
      | STATS COUNT_DISTINCT(user.name)`;
```